smithy-jsonschema/src/main/java/software/amazon/smithy/jsonschema/JsonSchemaConfig.java

```diff
@@ -403,4 +404,24 @@ public boolean getUseIntegerType() {
     public void setUseIntegerType(boolean useIntegerType) {
         this.useIntegerType = useIntegerType;
     }
+
+    /**
+     * Set the JSON schema version to use when converting Smithy shapes into Json Schema.
```

Comment:
nit: This looks looks like the text for the setter.

Reply:
updated in cr 3f12c5b.